app/controllers/admin/custom_emojis_controller.rb

```diff
@@ -32,23 +33,26 @@ def update
 
       if @custom_emoji.update(resource_params)
         log_action :update, @custom_emoji
-        redirect_to admin_custom_emojis_path, notice: I18n.t('admin.custom_emojis.updated_msg')
+        flash[:notice] = I18n.t('admin.custom_emojis.updated_msg')
       else
-        redirect_to admin_custom_emojis_path, notice: I18n.t('admin.custom_emojis.update_failed_msg')
+        flash[:alert] =  I18n.t('admin.custom_emojis.update_failed_msg')
       end
+      redirect_to admin_custom_emojis_path(page: params[:page], **@filter_params)
```

Comment:
rule of three says `admin_custom_emojis_path(page: params[:page], **@filter_params)` can probably be a helper? then you can just pass it a "notice" keyword and don't have to refactor to use `flash`

Reply:
-_-;

i suppose i could have the helper pass whatever is given (because sometimes we should alert rather than notice) 

what do you reckon such a helper should be called?

the reason i didn't make one is that it didn't feel like it would be a very meaningful method and i like to avoid increasing indirection if i can

Reply:
`current_filter_path`?